machine/alphanetworks/alphanetworks_snx60a0_486t/kernel/drivers-base-dd-alpha-snx60a0-486t.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/drivers/base/dd.c b/drivers/base/dd.c
+index 8a8d611..c2fa348 100644
+--- a/drivers/base/dd.c
++++ b/drivers/base/dd.c
+@@ -328,10 +328,15 @@ probe_failed:
+ 		if (local_trigger_count != atomic_read(&deferred_trigger_count))
+ 			driver_deferred_probe_trigger();
+ 	} else if (ret != -ENODEV && ret != -ENXIO) {
+-		/* driver matched but the probe failed */
+-		printk(KERN_WARNING
+-		       "%s: probe of %s failed with error %d\n",
+-		       drv->name, dev_name(dev), ret);
++		/* Skip the probe error message for these three devices on Alphanetworks platforms. */
++		if (strncmp(dev_name(dev), "0000:00:14.0", 12) &&
++			strncmp(dev_name(dev), "0000:00:14.2", 12) &&
++			strncmp(dev_name(dev), "0000:00:14.3", 12)) {
```

Comment:
These devices are Intel Rangeley GbE controllers. Only the 2nd GbE controller is used in our device and the other GbE controllers are not be used. Therefore, the error message will be displayed on console when the driver is trying to configure these unused GbE controllers. This piece of the code just mask the error message to prevent the users from confusing with these error messages.